packages/npm-dependencies-validation/.npmignore

```diff
@@ -0,0 +1,6 @@
+node_modules
```

Comment:
if you include the sources and the source maps
it will allow easier debugging for consumers.

Reply:
Also I am not sure `.npmignore` is needed
could we instead just specify the files we want to package **explicitly** in the package.json "files" section?

Reply:
Done